docs/api-reference/api.md

```diff
@@ -43,6 +43,16 @@ __Arguments__
 * __top_hash (str)__:  Optional. A package hash to delete, instead of the whole package.
 
 
+## get\_boto3\_session(\*, fallback: bool = True) -> boto3.session.Session  {#get\_boto3\_session}
+
+Return a Boto3 session with Quilt stack credentials and AWS region.
+In case of no Quilt credentials found, return a "normal" Boto3 session if `fallback` is `True`,
```

Comment:
```suggestion
In case if no Quilt credentials found, return a "normal" Boto3 session if `fallback` is `True`,
```

Reply:
AFAICT "In case of" is correct English (actually it's written by copilot)
default value is specified in signature